spec/controllers/elections/candidates_controller_spec.rb

```diff
@@ -1,41 +1,20 @@
 require 'rails_helper'
-
 RSpec.describe Elections::CandidatesController, type: :controller do
-  # Build an election
   let(:election) { create(:election) }
   before(:each) do
     allow(Election).to receive(:current).and_return(election)
   end
 
-  # Testing objects
   let(:user) { create(:user) }
-  let(:search_post) { create(:post) }
+  let(:search_post) { create(:post, elected_by: "Styrelsen") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.